### DISCUSSION TITLE
Minimum cost

### DISCUSSION THREAD

**Discussion by ugintl** [chaskiq/chaskiq#546] [Q&A] [answered]
What will be the minimum cost for production environment on heroku? Give some examples.

**Comment by michelson** [comment, marked as answer]
I would say that Heroku is not the most cost-effective. A good option could be digital ocean apps. You can spin a chaskiq instance (web/worker) for ~14 USD. You will also need a Postgres and Redis That could fit on another instance 7USD instance or pay their cluster service both services $30USD
Caprover is a good alternative too in terms of costs. Since you can spin one instance with all the services in one machine, a good machine for $40 USD dollars will run great.

**Reply by ugintl** [reply]
$40/month?

**Reply by michelson** [reply]
Hi, there is a digital ocean machine with these specs:

$48/mo $0.07143/hr
Memory 8GB
vCPUs 4 Intel CPUs
SSD 160GB NVMe SSDs

**Reply by ugintl** [reply]
A little expensive

**Reply by michelson** [reply]
well, Hetzner cloud has the same machine for less than 10 euros

**Reply by ugintl** [reply]
That would be around 11 USD. Thanks for the info. Are you aware of any free hosting where I can test a web app built on MERN stack?

**Reply by ugintl** [reply]
except heroku

**Reply by michelson** [reply]
try fly.io or vercel

**Reply by TurkerTunali** [reply]
We can run entire ERP system for 3 users in within a 1GB memory spec in digital ocean. Do we really need 8GB ?

**Comment by michelson** [comment]
No you don't. You can run chaskiq on heroku for free or on a hobby machines
14 usd

**Reply by psschand16** [reply]
how about running on a linux vm with docker?

**Reply by michelson** [reply]
what about it? you can run the app in a 2gb (web/worker) or in two instances 1gb each (web/worker) all will depends on the load and the number of users
